metricbeat/module/kubernetes/container/container_test.go

```diff
@@ -34,49 +34,65 @@ import (
 
 const testFile = "../_meta/test/stats_summary.json"
```

Comment:
Can you put a comment here that this is the file we are comparing with?
It is not obvious with first parse